packages/vuetify/src/components/VTimeline/VTimelineDivider.tsx

```diff
@@ -0,0 +1,85 @@
+// Components
+import { VIcon } from '@/components'
+import { VTimelineSymbol } from './VTimeline'
+
+// Composables
+import { useBackgroundColor } from '@/composables/color'
+import { makeElevationProps, useElevation } from '@/composables/elevation'
+import { makeSizeProps, useSize } from '@/composables/size'
+import { makeRoundedProps, useRounded } from '@/composables/rounded'
+
+// Utilities
+import { defineComponent, inject, toRef } from 'vue'
+
+export default defineComponent({
+  name: 'VTimelineDivider',
+
+  props: {
+    hideDot: Boolean,
+    lineColor: String,
+    icon: String,
+    iconColor: String,
+    fillDot: Boolean,
```

Comment:
You changed default look to be filled dot, but this prop still exist?

Reply:
What does the default dot look like? Is it just white with the elevation?